frontend/packages/dev-console/src/components/topology/componentUtils.ts

```diff
@@ -33,11 +34,18 @@ type EdgeProps = {
 };
 
 const MOVE_CONNECTOR_DROP_TYPE = '#moveConnector#';
+const MOVE_EV_SRC_CONNECTOR_DROP_TYPE = '#moveEvSrcConnector#';
 
 const MOVE_CONNECTOR_OPERATION = 'moveconnector';
 const REGROUP_OPERATION = 'regroup';
+const MOVE_EV_SRC_CONNECTOR_OPERATION = 'moveEvSrcConnector';
```

Comment:
You shouldn't need a separate operation for moving an event source connector. Similarly not needed for moving a connector in general. I think the original implementation which you used as a guide used operations incorrectly.

Operations are used to distinguish between differences in the same drag event when the user holds down keys to mutate the operation.

In this case there's only ever a single operation during move and therefore the default operation is sufficient.

Reply:
Have removed operation MOVE_EV_SRC_CONNECTOR_OPERATION and just using type